orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/tasks/SaveServiceAccountTask.java

```diff
@@ -135,11 +135,7 @@ private String generateSvcAcctName(Map<String, Object> pipeline) {
     if (pipeline.containsKey("serviceAccount")) {
       return (String) pipeline.get("serviceAccount");
     }
-
-    String pipelineName = (String) pipeline.get("name");
```

Comment:
Could also use application name + pipeline name

Reply:
id makes more sense since that way the service account is not orphaned when a pipeline is renamed.